File: reverseStringInt.java

public class reverseStringInt {
    public static void main(String[] args) {
        int num = 121;
        String rev = "";
        String number = Integer.toString(num);
        for(int i = number.length()-1; i>=0; i--){
            char c = number.charAt(i);
            rev+=c;

        }
        int finalrev = Integer.parseInt(rev);
        System.out.println(finalrev);

    }
}

File: reverseStringInt.go
package main

import (
	"fmt"
	"strconv"
)

func main() {
	var num int = 121
	var number string = strconv.Itoa(num)
	var rev string = ""
	for i := len(number) - 1; i >= 0; i-- {
		rev += string(number[i])
	}
	actual, _ := strconv.Atoi(rev)
	fmt.Println(actual)
}
